src/main/java/duckling/responses/ResponseHeaders.java

```diff
@@ -4,43 +4,65 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.stream.Collectors;
 
 public class ResponseHeaders {
     protected ResponseCode responseCode;
     protected String contentType;
+    protected String location;
     protected ArrayList<String> permittedMethods;
 
     public ResponseHeaders() {
-        this(ResponseCode.ok(), "null");
+        this(ResponseCode.ok(), "null", null);
     }
 
     public ResponseHeaders(ResponseCode responseCode) {
-        this(responseCode, "null");
+        this(responseCode, "null", null);
     }
 
-    public ResponseHeaders(ResponseCode responseCode, String contentType) {
+    public ResponseHeaders(
+        ResponseCode responseCode,
+        String contentType
+    ) {
+        this(
+            responseCode,
+            contentType,
+            null,
+            new ArrayList<>()
+        );
+    }
+
+    public ResponseHeaders(
+        ResponseCode responseCode,
+        String contentType,
+        String location
+    ) {
         this(
             responseCode,
             contentType,
+            location,
             new ArrayList<>()
         );
     }
 
     public ResponseHeaders(
         ResponseCode responseCode,
         String contentType,
+        String location,
         ArrayList<String> permittedMethods
     ) {
         this.contentType = contentType;
         this.responseCode = responseCode;
+        this.location = location;
```

Comment:
I'm a bit worried that you're going to have a zillion constructors for `ResponseHeaders`, every time you add a feature around responses. Not a blocker but the next time you're inclined to add something, think about some alternatives.